engines/director/lingo/xlibs/applecdxobj.h

```diff
@@ -22,11 +22,20 @@
 #ifndef DIRECTOR_LINGO_XLIBS_APPLECDXOBJ_H
 #define DIRECTOR_LINGO_XLIBS_APPLECDXOBJ_H
 
+#include "common/formats/cue.h"
```

Comment:
I'd recommend forward declaration for `Common::CueSheet` instead of the whole #include it in the header.

Reply:
Done!